adding travel grant information (#46)
USRSE/usrse24: pages/attend/travel.md

- Hotel Andaluz Albuquerque, Curio Collection - 330 Tijeras Ave NW, Albuquerque, NM 87102 (0.1 miles away)

## Travel Grants

> The US-RSE ’24 conference will again be offering travel grants.

Reviewer: ```suggestion
The US-RSE'24 conference will again be offering travel grants via an application process.
```

> The US-RSE ’24 conference will again be offering travel grants via an application process.

Reviewer 2: ```suggestion
The US-RSE’24 conference will again be offering travel grants via an application process.
```

- Travel grants can be used for transportation, registration, and/or accommodations.
- Meals will be provided at the conference on conference days.
- Priority will be given to...

>   - those having an accepted submission at the US-RSE’24 conference.

Reviewer 3: I have a nitpick: do we want "US-RSE’24" or "US-RSE ’24" (with/without the space)? Right now we have one of each.

Reviewer 4: I vote no space. Other pages (as well as the header) have "US-RSE’24".

Reviewer 5: no space is what we've used generally

Reviewer 2: Based on consensus, I removed the space from the one "US-RSE" that had it.

> - Priority will be given...
>   - to those having an accepted submission at the US-RSE’24 conference.
>   - to enhance the diversity of participants.

Reviewer: ```suggestion
- Priority will be given to...
  - those with an accepted submission at the US-RSE’24 conference.
  - enhance the diversity of participants.
```
